R/authentication.R

```diff
@@ -252,11 +254,6 @@ AbstractOIDCAuthentication <- R6Class(
         private$scopes = list("openid")
       } else {
         private$scopes = provider$scopes
-        
-        #TODO remove later, this is used for automatic reconnect
-        if (!"offline_access" %in% private$scopes) {
-          private$scopes = c(private$scopes, "offline_access")
-        }
```

Comment:
Not sure about this, I've asked @soxofaan how it's handled in Python.

Reply:
>  The python client indeed automatically adds "offline_access" scope if it is supported by the OIDC provider (based on its OIDC config at .well-known/openid-configuration):
https://github.com/Open-EO/openeo-python-client/blob/2df4d514fc3bad112482cfe3f1a9b93d4646e288/openeo/rest/auth/oidc.py#L290-L302
but as you see this is only done if we actually want refresh tokens.
> 
> For the client credentials, we don't need refresh tokens, so the logic to add "offline_access" is not triggered